RIOT-OS/riot-os.org
# Add cookies consent alert

## Contribution description
This adds an alert for the user to consent the usage of third party cookies, and the mechanism to keep track of that.
This now delays the load of the Twitter script until cookies are accepted.

For a reference see https://gdpr.eu/cookies/

### What it does
- If the user accepts a preference cookie is stored with the value `true` for 12 months. Then, an event is triggered. Currently this only loads Twitter's script.
- If the user rejects a preference cookie is stored with the value `false` for 12 months. Then an event is triggered.
- Once the user has expressed the decision the alert is not shown again.

### How to test
1. Clear all the cookies first
2. Try both accepting and rejecting. In both cases you should see the preference cookie. When accepted the Twitter timeline should load and you'll see Twitter's cookies as well. Make sure you refresh the cookies list to watch for changes.
